### ISSUE TITLE
Add traditional ASP.NET routing

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#9]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [x] Developer

### Needs what

Conventional/attribute ASP.NET routing. As an alternative development path in the course module. The code of the traditional routing will be commented out in the final codebase.

### Why

Because this is an enterprise-grade approach, as opposed to #8 .

### Issue expands upon

This issue expands upon another issue(s): #8

### Blockers

This issue is blocked by the following issue(s): #15
